src/components/Header/Header.tsx

```diff
@@ -0,0 +1,14 @@
+import { Typography, Stack } from "@mui/material";
+import { HeaderNav } from "./HeaderNav";
+
+export const Header = () => {
+  return (
+    <Stack direction="row" width="1100px" alignItems="center" justifyContent="space-between" padding="14px 19px">
```

Comment:
What about mobile version?
width="1100px" is a lot for the mobile. So, you can make it with the useMedia if I am not mistaken in the MUI

Reply:
done